rpc-server/src/modules/blocks/mod.rs

```diff
@@ -295,16 +295,31 @@ impl BlocksInfoByFinality {
         let final_block_future = crate::utils::get_final_block(near_rpc_client, false);
         let optimistic_block_future = crate::utils::get_final_block(near_rpc_client, true);
         let validators_future = crate::utils::get_current_validators(near_rpc_client);
-        let (final_block, optimistic_block, validators) = tokio::try_join!(
+        let (final_block, optimistic_block, validators) = futures::future::join3(
```

Comment:
I'd like to simplify this more, but I'm out of ideas. They have different return types, so I can't use regular `join_all`. Please, let me know if you have any ideas

Reply:
This is done once when the service starts. Any drop at this point should not prevent the service from starting. So I don't think we need to guarantee that all tasks are completed here.